src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx

```diff
@@ -234,14 +202,116 @@ These Support Plans apply only to your paid service subscription under an existi
           </td>
         </tr>
       </tbody>
-    </table>
+</table>
+
+### Response times 
+
+The table below shows estimated response times based on your New Relic usage plan. 
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="2"></th>
+            <th colspan="3">**First response target**</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                Priority
+            </td>
+            <td>
+                Definition
+            </td>
+            <td>
+                Standard
+            </td>
+            <td>
+                Pro
+            </td>
+            <td>
+                Enterprise
+            </td>
+        </tr>
+        <tr>
+            <td>
+                P1
+            </td>
+            <td>
+                * A majority of New Relic platform functionality is down or instrumentation is not reporting data. 
+                * Observability is completely unavailable for an existing business-critical application. No workaround is available. 
+            </td>
+            <td>
+                2 business days
```

Comment:
@homelessbirds I wonder if we need to follow the style guide here and spell out the days under 10. What do you think? For example, "Two hours," etc..

Reply:
I had discussed this with the team and we decided on keeping them numerical to match other examples of this sort of doc.